engine/src/main/scala/cromwell/webservice/WebServiceUtils.scala

```diff
@@ -32,6 +32,15 @@ trait WebServiceUtils {
     complete((statusCode, warningHeaders(warnings), value))
   }
 
+  // 2.13 added this, not sure why the baseline version was compiling actually.
```

Comment:
what the comment said... I tested this manually for batch submits and it appears to work fine in 2.13 with these changes